config/cass-operator/ns-scoped/kustomization.yaml

```diff
@@ -3,38 +3,10 @@ kind: Kustomization
 
 resources:
 - github.com/k8ssandra/cass-operator/config/deployments/default?ref=master
-patches:
-- target:
-    kind: ConfigMap
-    name: cass-operator-manager-config
-  patch: |
-    - op: replace 
-      path: /data/image_config.yaml
-      value: |
-        # Your custom config goes here. Please refer to `./config/manager/image_config.yaml` for an example
-        # of what fields and configurations are available.
-        apiVersion: config.k8ssandra.io/v1beta1
-        kind: ImageConfig
-        metadata:
-          name: image-config
-        images:
-          system-logger: "k8ssandra/system-logger:latest"
-          config-builder: "datastax/cass-config-builder:1.0.4-ubi7"
-          # cassandra:
-          #   "4.0.0": "k8ssandra/cassandra-ubi:latest"
-          # dse:
-          #   "6.8.999": "datastax/dse-server-prototype:latest"
-        # imageRegistry: "localhost:5000"
-        # imagePullPolicy: Always
-        # imagePullSecret:
-        #   name: my-secret-pull-registry
-        defaults:
-          # Note, postfix is ignored if repository is not set
-          cassandra:
-            repository: "k8ssandra/cass-management-api"
-          dse:
-            repository: "datastax/dse-mgmtapi-6_8"
-            suffix: ""
+
 images:
   - name: k8ssandra/cass-operator
     newTag: v1.12.0
+
+components:
```

Comment:
Using the component makes it possible to reuse the ImageConfig patch across the cass-operator overlays. To do our due diligence we should also make sure it is possible for someone using k8ssandra-operator to patch and change the ImageConfig.